python/lsst/summit/extras/slewTimingSimonyi.py

```diff
@@ -0,0 +1,351 @@
+    prePadding : `float`, optional
+        The amount of time to pad before the start of the first exposure.
+    postPadding : `float`, optional
+        The amount of time to pad after the end of the last exposure.
+
+    Returns
+    -------
+    fig : `matplotlib.figure.Figure`
+        The figure containing the plot.
+    """
+    inPositionAlpha = 0.5
+    commandAlpha = 0.5
+    integrationColor = "grey"
+    readoutColor = "blue"
+
+    legendHandles = []
+
+    expRecords = sorted(expRecords, key=lambda x: (x.day_obs, x.seq_num))  # ensure we're sorted
+
+    startSeqNum = expRecords[0].seq_num
+    endSeqNum = expRecords[-1].seq_num
+    dayObs = expRecords[0].day_obs
+    title = f"Mount command timings for {dayObsIntToString(dayObs)} seqNums {startSeqNum} - {endSeqNum}"
+
+    begin = expRecords[0].timespan.begin
+    end = expRecords[-1].timespan.end
+
+    az, el, rot, _ = getAzElRotDataForPeriod(client, begin, end, prePadding, postPadding)
+
+    # Create a figure with a grid specification and have axes share x
+    # and have no room between each
+    fig = plt.figure(figsize=(12, 6))
+    gs = fig.add_gridspec(3, 1, hspace=0)
+    azimuth_ax = fig.add_subplot(gs[0, 0])
+    elevation_ax = fig.add_subplot(gs[1, 0], sharex=azimuth_ax)
+    rotation_ax = fig.add_subplot(gs[2, 0], sharex=azimuth_ax)
+    axes = {"az": azimuth_ax, "el": elevation_ax, "rot": rotation_ax}
+
+    # plot the telemetry
+    axes["az"].plot(az["actualPosition"])
+    axes["el"].plot(el["actualPosition"])
+    axes["rot"].plot(rot["actualPosition"])
+
+    # shade the expRecords' regions including the readout time
+    for i, record in enumerate(expRecords):
+        # these need to be in UTC because matplotlib magic turns all the axis
+        # timings into UTC when plotting from a dataframe.
+        startExposing = record.timespan.begin.utc.datetime
+        endExposing = record.timespan.end.utc.datetime
+
+        readoutEnd = (record.timespan.end + READOUT_TIME).utc.to_value("isot")
+        seqNum = record.seq_num
+        for axName, ax in axes.items():
+            ax.axvspan(startExposing, endExposing, color=integrationColor, alpha=0.3)
+            ax.axvspan(endExposing, readoutEnd, color=readoutColor, alpha=0.1)
+            if axName == "el":  # only add seqNum annotation to bottom axis
+                label = f"seqNum = {seqNum}"
+                midpoint = startExposing + (endExposing - startExposing) / 2
+                ax.annotate(
+                    label,
+                    xy=(midpoint, 0.5),
+                    xycoords=("data", "axes fraction"),
+                    ha="center",
+                    va="center",
+                    fontsize=10,
+                    color="black",
+                )
+
+    # place vertical lines at the times when axes transition in/out of position
+    for label, topic in inPositionTopics.items():
+        # TODO: need to iterate over colours in this loop
```

Comment:
Just flagging this in case you had intended to do it on this branch.

Reply:
Thanks - I don't remember fixing this, so I guess it got fixed in a rebase/refactor.